### PR TITLE
[TRAFODION-2855] Correct the syntax descriptions of LOAD Statement for *Trafodion SQL Reference Manual* 2

### PR DESCRIPTION
**Pull request by liuyu000**

### CODE REVIEW

File: docs/sql_reference/src/asciidoc/_chapters/sql_utilities.adoc [hunk 1]
```diff
@@ -443,24 +443,39 @@ specify one or more of these options:
...
 If error rows must be written to a specified location, then specify TO _error-location-name_, otherwise they will be written to the default location.
+`_error-location-name_` must be a HDFS directory name to which trafodion has write access.
 +
-Error logs are written in separate files by the processes involved in the load command under sub-directory representing the load command in the given location.
-The actual log file location is displayed in the load command output.
+Error rows will be logged in HDFS files in the *directory* `/user/trafodion/bulkload/logs` if the error log location is not specified. 
++
+The default name of the *subdirectory* is `_ERR_catalog.schema.target_table_date_id_`, where `_id_` is a numeric identifier timestamp (YYYYMMDD_HHMMSS) unique to the process where the error was seen.
++
+The default name of the *error file* is `_loggingFileNamePrefix_catalog.schema.target_table_instanceID_`, where `_loggingFileNamePrefix_` is hive_scan_err or traf_upsert_err depending on the data source table, and `_instanceID_` is the ID of instance starting from 0, generally there is only one instance.
```
Comment (DaveBirdsall) [inline docs/sql_reference/src/asciidoc/_chapters/sql_utilities.adoc]: Suggest "...is the instance ID starting from 0, ..."

File: docs/sql_reference/src/asciidoc/_chapters/sql_utilities.adoc [hunk 2]
```diff
@@ -443,24 +443,39 @@ specify one or more of these options:
...
++
+3. `traf_upsert_err_TRAFODION.SEABASE.TEST_LOAD_LOG_0` is the default name of *error file*.
 
-*** `_error-location-name_`
+*** Error logs
++
+Error logs are written in separate files by the processes involved in the load command under sub-directory representing the load command in the given location.
 +
-must be a HDFS directory name to which trafodion has write access.
+The actual log file location is displayed in the load command output. It is recommended that use the same location for load as it’s easier to find the error logs.
```
Comment (DaveBirdsall) [inline docs/sql_reference/src/asciidoc/_chapters/sql_utilities.adoc]: Suggest "It is recommended that you use..." (add the word "you")

File: docs/sql_reference/src/asciidoc/_chapters/sql_utilities.adoc (hunk 1 above)
Comment (liuyu000) [reply docs/sql_reference/src/asciidoc/_chapters/sql_utilities.adoc]: OK, thanks Dave :)

File: docs/sql_reference/src/asciidoc/_chapters/sql_utilities.adoc (hunk 2 above)
Comment (liuyu000) [reply docs/sql_reference/src/asciidoc/_chapters/sql_utilities.adoc]: OK, thanks for your eagle eye, Dave :)

**Pull request merged**